modules/tpm_filter.nf

```diff
@@ -5,8 +5,13 @@ process tpm_filter {
     label 'python3'
     label 'smallTask'
 
-    if (params.cloudProcess) { publishDir "${params.output}/${params.tpm_filter_dir}", mode: 'copy', pattern: "**.counts.filtered.formated.tsv" }
-    else { publishDir "${params.output}/${params.tpm_filter_dir}", pattern: "**.counts.filtered.formated.tsv" }
+    if (params.cloudProcess) { 
```

Comment:
Do we need the distinction between 'cloud' and 'local' here? When it's local it will be simply linked and not copied, right?

Reply:
what we want is: copy if cloud, and link if local.

I'm not sure, if it does this without the if clause.

Reply:
but the count files are small, so we can just copy them always?

Reply:
I'd handle that always in the same way

Reply:
Consistency! Fine with me!